src/connection.js

```diff
@@ -5,6 +5,7 @@ const pull = require('pull-stream')
 const Pushable = require('pull-pushable')
 
 const PROTOCOL = require('./protocol')
+const UTILS = require('./utils')
```

Comment:
No need for uppercase here, I think. (the previous uppercase is because it's a string constant).

Reply:
```
const utils = require('./utils') 
``` 
it's ok for you ?

Reply:
ok, but see the file name change below.